packages/chats/src/Lightbox/Duplicate.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import type { DuplicateProps } from './types';
+import { DuplicateContent } from './DuplicateContent';
+
+export const Duplicate = (props: DuplicateProps) => {
```

Comment:
Do we really need the wrapper, maybe do it in `Lightbox`?

Reply:
You're right. I removed `Duplicate`, and renamed `DuplicateContent` to `Duplicate`

Reply:
I changed my mind. This wrapper is also needed to remove the Portal from the DuplicateContent

Reply:
Perfect now! Thank you :)